src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is running");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer is stopped");
```

Comment:
The message 'Bulldozer is stopped' should be more informative. It would be better to indicate what kind of work has been stopped.